### ISSUE TITLE
Unable to change the Accessibility Description for menu Overlay Icon for toolbar in Android

### ISSUE THREAD

**Issue by Alekhya03** [material-components/material-components-android#3115]
Hi Team,
      I am trying to set a new description to the overlay Icon(3 dots button in the toolbar) but it's not replicating. can you help me with this?

**Comment by drchen** [comment]
I'll close this as it's a duplicate to #3116
